events/2023-06-20-S4PST.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: S4PST: Sustainability for Programming Systems and Tools 
+description: U.S. Department of Energy (DOE) <br /> 
+Office of Advanced Scientific Computing Research (ASCR) <br />
+permalink: /events/s4pst2023/
```

Comment:
Should we use s4pst-May-2023 to distinguish from a future S4PST workshop?

Reply:
As this is the website of the project, I think that it is OK.